spark3-extensions/src/main/antlr/org.apache.spark.sql.catalyst.parser.extensions/IcebergSqlExtensions.g4

```diff
@@ -156,14 +169,17 @@ ASC: 'ASC';
 BY: 'BY';
 CALL: 'CALL';
 DESC: 'DESC';
+DISTRIBUTED: 'DISTRIBUTED';
```

Comment:
These also need to be added to `nonReserved`, right?

Reply:
Yep, you are right.

Reply:
Added.